llvm/lib/Transforms/Scalar/LICM.cpp

```diff
@@ -2808,6 +2808,7 @@ static bool hoistMulAddAssociation(Instruction &I, Loop &L,
     auto *NewBO =
         BinaryOperator::Create(Ins->getOpcode(), LHS, RHS,
                                Ins->getName() + ".reass", Ins->getIterator());
+    NewBO->setDebugLoc(DebugLoc::getDropped());
```

Comment:
Previous comment said

> it should be the other way around (i.e. this uses getDropped() and the above keeps using dropLocation()

"the above" referring to this one here, I think? maybe I'm confused about which we're talking about now though (thanks github for evaporating my comments)

Reply:
There was an instance of `dropLocation` that got replaced by `getDropped` in this patch - I've just fixed that, such that we're now only adding `getDropped` and not adding or removing any `dropLocation`s!